### ISSUE TITLE
Mediterranean example doesn't seem to use `ECCORestoring` correctly

### ISSUE THREAD

**Issue by glwagner** [CliMA/ClimaOcean.jl#221]
Unless I am misunderstanding something, this seems wrong:

https://github.com/CliMA/ClimaOcean.jl/blob/b19c13bc8e3f5a0e9c5ab3f7464ed3d56ff8e9d0/examples/mediterranean_simulation_with_ecco_restoring.jl#L81-L82

I'm not sure if there is another example of how to use `ECCORestoring` lying around?

**Comment by glwagner** [comment]
@simone-silvestri

**Comment by waywardpidgeon** [comment]
Yes, in main the function ECCO_restoring_forcing is missing from ECCO_restoring.jl and the inlined ECCORestoring has completely different arguments which need to be restructured. Maybe someone meant to write the former function or has already done this and not yet included it? 

In the examples subdirectory the call occurs in the mediterranean_simulation_with_ecco_restoring.jl but also in two other cases,both in mesoscale_resolving_omip directory.

**Comment by glwagner** [comment]
What happened is that the source code was changed, but the example was not updated. We're still working on getting all the examples to be tested; this has been a slow process due to the computationally intensive nature of the examples. cc @navidcy

**Comment by navidcy** [comment]
The Mediterranean example is not part of the docs yet so it's not being run every time the docs are built.
